addons/website_slides/controllers/mail.py

```diff
@@ -13,30 +13,36 @@
 
 class SlidesPortalChatter(PortalChatter):
 
-    @http.route(['/mail/chatter_post'], type='http', methods=['POST'], auth='public', website=True)
+    @http.route(['/mail/chatter_post'], type='json', methods=['POST'], auth='public', website=True)
     def portal_chatter_post(self, res_model, res_id, message, **kw):
         result = super(SlidesPortalChatter, self).portal_chatter_post(res_model, res_id, message, **kw)
+        attachments = request.env['mail.message'].browse(result['default_message_id']).sudo().attachment_ids
+        result.update({'default_attachment_ids': attachments.read(['id', 'name', 'mimetype', 'file_size', 'access_token'])})
```

Comment:
This `sudo` is not acceptable as it allows anyone to read all attachment linked to any message.

Reply:
Removed as per your suggestion.